### PR TITLE
Intruduction of getfee command, fixes #3704

### LINKED ISSUE #3704: RFC: new command: getfee
# Request For Comments

### Problem

When you use Electrum as a merchant, you want to suggest your customers a correct fee to avoid problems. Electrum already has this information, why then use an another external source to create another point of failure?

### Proposed Solution

Creation of another command ```getfee``` - which will operate via RPC as well. It's code will be identical to currently displayed fee when you issue a command line ```electrum daemon status```.

### Who will write it

If concept is agreed, I will propose #PR for this.

### PR DESCRIPTION
**Pull request by ser**

### CODE REVIEW

File: lib/commands.py [hunk 1]
```diff
@@ -672,6 +672,12 @@ def is_synchronized(self):
         """ return wallet synchronization status """
         return self.wallet.is_up_to_date()
 
+    @command('')
```
Comment (bauerj) [inline lib/commands.py]: Shouldn't this use `'n'` instead?

Comment (ser) [reply lib/commands.py]: Fixed.

**Comment by ecdsa** [comment]
can you rename it "getfeerate" ?

**Comment by ecdsa** [comment]
I had to merge manually because commits were not rebased

**Comment by ser** [comment]
Sorry, next time I will remember.

**Pull request closed without merging**